test/bun.js/websocket-server.test.ts

```diff
@@ -138,8 +138,7 @@ describe("websocket server", () => {
           if (
             server.upgrade(req, {
               data: "hello world",
-
-              headers: 1238,
+              headers: 1238 as any, // why is this a number?
```

Comment:
we are testing that numbers/invalid types don't cause a crash

we will need a lot of tests like that tbh